akka-projection-core/src/main/scala/akka/projection/provider/scaladsl/SourceProvider.scala

```diff
@@ -2,7 +2,7 @@
  * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
  */
 
-package akka.projection.scaladsl
+package akka.projection.provider.scaladsl
```

Comment:
I'm in doubt about the names of package here. It's not a `provider`, but `source provider`, but `akka.projection.source.provider` for package sounds weird to me.